File: engine/engine-model/src/main/scala/tools/packet/guild/ResponseGuildMessage.scala
package tools.packet.guild

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ResponseGuildMessage(private var _code: Byte, private var _targetName: String) extends PacketInput {
  def code: Byte = _code

  def targetName: String = _targetName

  override def opcode(): SendOpcode = SendOpcode.GUILD_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/guild/ResponseGuildMessage.java
package tools.packet.guild;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ResponseGuildMessage(Byte code, String targetName) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.GUILD_OPERATION;
   }
}